src/components/Tags/_tags.scss

```diff
@@ -0,0 +1,134 @@
+@use '#scss/allPartials' as *;
+
+.#{$px}-tags-list {
+  display: flex;
+  flex-flow: row wrap;
+  gap: $spacing-sm;
+
+  .#{$px}-tag {
+    align-items: center;
+    background-color: $pure-white;
+    border: 1px solid $button-hover;
+    border-radius: 6.25rem;
+    cursor: default;
+    display: flex;
+    gap: 0;
+    height: $button-tertiary-height;
+    justify-content: center;
+    padding: 0 $spacing-xsm 0 $spacing-sm;
+
+    &__label {
+      color: $button-hover;
+      font-size: 0.75rem;
+      font-variation-settings: 'wght' 600;
+      text-align: center;
+    }
+
+    .#{$px}-close-button {
+      align-items: center;
+      display: flex;
+      gap: $spacing-micro;
+      height: $button-tertiary-height;
+      justify-content: center;
+      padding: 0;
+
+      &__close {
+        font-size: 0.75rem;
+        height: 1.65rem;
+        width: 1.65rem;
+
+        svg {
+          path {
+            fill: $button-hover;
+          }
+
+          &:focus {
+            fill: $pure-black;
+          }
+        }
+
+        &:hover {
+          background-color: $pure-white;
+
+          svg {
+            path {
+              fill: $pure-black;
+            }
+          }
+
+        }
+
+        &:focus {
+          svg {
+            path {
+              fill: $pure-black;
+            }
+          }
+        }
+      }
+    }
+
+  }
+
+  &--clear {
+    border-color: $pure-white;
+    display: flex;
+    gap: 0;
+    height: 2rem;
+
+    &:hover {
+      cursor: pointer;
+
+      .#{$px}-left-arrow {
+        svg {
+          fill: $pure-black;
+
+          path {
+            fill: $pure-black;
+          }
+        }
+      }
+    }
+
+    &:focus-visible {
+      border-color: $button-hover;
+      border-radius: 6.25rem;
+      gap: 0;
+      outline-offset: 0;
+      padding: 0;
+    }
+
+    .#{$px}-left-arrow {
+      height: 1.25rem;
```

Comment:
In the figma the icon size is 16px, but it's using some kind of weird snowflake token var: `--design-cheats-for-button-small-icon-size-11`.  We need to check with @RyanOlanoPHILLIPS to see how this scales at different breakpoints.  I assume it's not supposed to stay 16 the whole time